capstone-app/src/app/modules/onboarding/content/content.component.scss

```diff
@@ -7,20 +7,8 @@ img {
     height: 400px;
 }
 
-@media only screen and (max-width: 600px) {
-    mat-panel-title ::ng-deep {
-        font-size: 12px;
-        height: fit-content;
-        margin: 0;
-    }
-
-    mat-panel-description ::ng-deep {
-        font-size: 12px;
-        height: fit-content;
-        margin: 0;
-    }
-    
-    mat-expansion-panel p {
-        font-size: 12px;
+@media only screen and (min-width: 700px) and (max-width: 1000px) {
+    img {
```

Comment:
Recommend using a more specific selector than "img". Add a class to the html img if necessary to reference here.